jps-plugin/src/org/jetbrains/jps/incremental/scala/data/HydraData.scala

```diff
@@ -0,0 +1,36 @@
+package org.jetbrains.jps.incremental.scala.data
+
+import java.io.File
+
+import org.jetbrains.jps.incremental.scala.SettingsManager
+import org.jetbrains.jps.model.JpsProject
+
+import scala.collection.JavaConverters._
+
+/**
+  * @author Maris Alexandru
+  */
+case class HydraData(project: JpsProject, files: Map[String, List[File]]) {
+  def getCompilerJar(scalaVersion: String): Option[File] = {
+    files.get(scalaVersion).getOrElse(List()).find(_.getName.matches(s".*scala-compiler-$scalaVersion-hydra\\d\\d\\.jar"))
```

Comment:
Can't we use `\\d+` instead of `\\d\\d\\`? (this way the code also won't break if we get to release scala-hydra100 :)

Reply:
Also, you could extract the regular expression and avoid the pattern is recompiled on every try to match